test/com/amazon/ion/impl/IonReaderContinuableTopLevelBinaryTest.java

```diff
@@ -234,10 +220,11 @@ private static byte[] toBinary(String ion) {
      * Creates an incremental reader over the given binary Ion, constructing a reader either from byte array or
      * from InputStream depending on the value of the parameter 'constructFromBytes'.
      * @param builder the reader builder.
+     * @param constructFromBytes whether to construct the reader from bytes or an InputStream.
      * @param bytes the binary Ion data.
      * @return a new reader.
      */
-    private IonReader readerFor(IonReaderBuilder builder, byte[] bytes) {
```

Comment:
Scrolling back a ways to reach this method, my only quibble in this PR so far is the name of the parameter `constructFromBytes`. Given a `byte[] bytes` it seems natural that requested reader would be constructed from it, so requiring a boolean `true` for this state of affairs seems odd. But it gets weirder when `constructFomBytes == false`. `true` I understand, but what will `false` do?

The documentation for this parameter "whether to construct the reader from  bytes or an InputStream" clears that up, but it makes me wonder whether this parameter to be `wrapBytesWithInputStream` 🤷‍♂️.